app/navigation/Navigation.scala

```diff
@@ -235,6 +236,7 @@ class Navigation @Inject()(dateTimeHelper: DateTimeHelper,
       case ReasonableExcuses.technicalIssues => controllers.routes.TechnicalIssuesReasonController.onPageLoadForWhenTechnologyIssuesBegan(mode)
       case ReasonableExcuses.health => controllers.routes.HealthReasonController.onPageLoadForWasHospitalStayRequired(mode)
       case ReasonableExcuses.other => controllers.routes.OtherReasonController.onPageLoadForWhenDidBecomeUnable(mode)
+      case _ => throw new MatchError(s"[Navigation][getNextURLBasedOnReasonableExcuse] - Unknown reasonable excuse controller ${controllers.routes.AppealAgainstObligationController.onPageLoad(mode)}")
```

Comment:
The wrapping code is not clear but this relates to the reasonable excuse, not the controller 😄 
```suggestion
      case _ => throw new MatchError(s"[Navigation][getNextURLBasedOnReasonableExcuse] - Unknown reasonable excuse $reasonableExcuse")
```